src/Unosquare.Swan.Lite/Formatters/Json.Converter.cs

```diff
@@ -102,12 +98,17 @@ private static void GetByteArray(string sourceString, ref object target)
                 } // Get the string bytes in UTF8
             }
 
-            private static object GetSourcePropertyValue(IDictionary<string, object> sourceProperties,
+            private string GetNameWithCase(string name) => _jsonSerializerCase == JsonSerializerCase.PascalCase
+                ? name
```

Comment:
```suggestion
                ? char.ToUpperInvariant(name[0]) + name.Substring(1);
```
To ensure you return PascalCase